tests/sqlparser_sqlite.rs

```diff
@@ -281,6 +282,38 @@ fn parse_create_table_gencol() {
     sqlite_and_generic().verified_stmt("CREATE TABLE t1 (a INT, b INT AS (a * 2) STORED)");
 }
 
+#[test]
+fn parse_create_table_on_conflict_col() {
+    for keyword in [
+        Keyword::ROLLBACK,
+        Keyword::ABORT,
+        Keyword::FAIL,
+        Keyword::IGNORE,
+        Keyword::REPLACE,
+    ] {
+        let sql = format!("CREATE TABLE t1 (a INT, b INT ON CONFLICT {:?})", keyword);
+        match sqlite().verified_stmt(&sql) {
```

Comment:
```suggestion
        match sqlite_and_generic().verified_stmt(&sql) {
```
just a heads up, in case the GenericDialect suggestion is applied, this bit changes as such

Reply:
I applied the commit, but without having the GenericDialect allowing the "ON CONFLICT" syntax this leads to an error in the tests, since parser righteously returns a ParserError needing of a comma:

```rust
ParserError("Expected: ',' or ')' after column definition, found: ON")
```
In case the GenericDialect was to allow such column option than a test case with
```
match sqlite_and_generic().verified_stmt(&sql) {
```
would pass, but at the moment it doesn't.

So this depends on extending the GenericDialect or not.